index.html

```diff
@@ -265,5 +265,43 @@ <h3>Skills</h3>
         </div>
       </section>
     </section>
+
+    <!-- Contact section -->
+    <footer class="contact">
+      <h1>Contact Me</h1>
+      <p>
+        If you have an application you are interested in developing, a feature
+        that you need built or a project that needs coding. I’d love to help
+        with it
+      </p>
+
+      <form action="https://formspree.io/f/mvolzpoz" method="post" id="my-form">
+        <input
+          type="text"
+          placeholder="Full name"
+          maxlength="30"
+          name="full-name"
+          required
+        />
```

Comment:
- [ ] I would recommend using `flexbox` to align elements inside form, that would be easier to change the view when it gets to desktop version since it's easier to manipulate flexbox and flex item